source/_components/switch.wirelesstag.markdown

```diff
@@ -0,0 +1,44 @@
+---
+layout: page
+title: "WirelessTag Switch"
+description: "Instructions on how to integrate your Wireless Tags sensors within Home Assistant."
+date: 2018-03-26 21:52
+comments: false
+sidebar: true
+sharing: true
+footer: true
+logo: wirelesstag.png
+ha_category: Switch
+ha_iot_class: "Local Push"
+---
```

Comment:
`ha_version` is missing.

Reply:
@frenck thanks! what version would you like to me to put here? as I'm not aware to what version  this contribution will go finally ;) so i left no version intentionally...

Reply:
General recommendation is to pick the next "major"  release, which is `0.67`.

Reply:
@frenck got it, thx! in progress with it...